main.js

```diff
@@ -0,0 +1,60 @@
+/* eslint max-classes-per-file: ["error", 2] */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+const bookList = document.getElementById('booklist');
+const addLi = (id, title, author) => {
+  const list = document.createElement('li');
+  list.id = id;
+  list.className = 'books-lists';
+  list.innerText = `${title} - ${author}`;
+  const button = document.createElement('button');
+  button.type = 'button';
+  button.innerText = 'Delete';
+  list.appendChild(button);
+  return list;
+};
+class AllBooks {
+  constructor() {
+    this.data = [];
+  }
+
+  addBook(title, author) {
+    this.data.push(new Book(title, author));
+    document.getElementById('Title').value = '';
+    document.getElementById('Author').value = '';
+    this.refresh();
+  }
+
```

Comment:
- [ ] As the previous reviewer asks for the data on the page is not showing when you refresh the page. 

![Capture](https://user-images.githubusercontent.com/64173189/134249216-d77fa9c2-d96d-4ce9-ac02-be090619799e.JPG)

Here there is everything working properly but once you refresh the page the data disappear from the page but still is in local storage. 

![Capture](https://user-images.githubusercontent.com/64173189/134249315-22edea86-c197-43b2-b7d4-e958e5989eef.JPG)

It would be better if you fix this minor issue. [Here](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md) is some refresh about local storage. 

🛠️ 😄